sidecar/setup.sh

```diff
@@ -1,15 +1,17 @@
 #!/bin/bash
+mc alias set sigma2 $S3_HOST $S3_ACCESS_KEY $S3_SECRET_KEY
+
 echo "Create the bucket if it doesn't exist"
 set +e
-s4cmd mb s3://$S3_BUCKET_NAME --endpoint-url $S3_HOST
+mc mb "sigma2/$S3_BUCKET_NAME"
```

Comment:
so should we try `/root/minio-binaries/mc mb "sigma2/$S3_BUCKET_NAME"` here instead?